ui/react-ui/src/components/routes/userTasks/UserTasks.js

```diff
@@ -28,9 +38,12 @@ const UserTasks = () => {
         ? <BootstrapTable heatherItems={Object.keys(userTasks[2][0])} rows={userTasks[2]} /> 
         : <h3>User {userId} Tasks Not Found</h3>
       }
+      <button onClick={this.deleteTask.bind(this)}>Delete Task</button>
```

Comment:
I think this should be like this =>
<button onClick={() => this.deleteTask()}>Delete Task</button>

Reply:
Fixed it. Thanks!